include/crm/common/results.h

```diff
@@ -273,6 +274,7 @@ typedef enum crm_exit_e {
     CRM_EX_NOT_YET_IN_EFFECT    = 111, //!< Requested item is not in effect
     CRM_EX_INDETERMINATE        = 112, //!< Could not determine status
     CRM_EX_UNSATISFIED          = 113, //!< Requested item does not satisfy constraints
+    CRM_EX_NO_DC                = 114, //!< DC is not yet elected, e.g. right after cluster restart
```

Comment:
One thing Ken missed is that whenever a value is added to `crm_exit_e`, `python/pacemaker/exitstatus.py` also needs to be updated.  That's fine - I'll merge this PR now, and submit a followup to add that value.

Reply:
@clumens A section in the Pacemaker Development doc for this seems like a good idea. Documentation updates are a pain I know, and we have a ton of other room for improvement there. I'm not consistent about updating them either.